src/lib.rs

```diff
@@ -142,6 +143,7 @@
 //! [20]: https://www.unix.com/man-page/mojave/4/random/
 //! [21]: https://www.freebsd.org/cgi/man.cgi?query=getrandom&manpath=FreeBSD+12.0-stable
 //! [22]: https://leaf.dragonflybsd.org/cgi/web-man?command=getrandom
+//! [23]: https://github.com/espressif/esp-idf/blob/master/components/esp_hw_support/hw_random.c
```

Comment:
As noted earlier by @josephlr it's better to link the relevant page from https://docs.espressif.com/, not the source code.

Reply:
I have updated the documentation link to reference the API documentation.